src/main/java/com/teamwizardry/librarianlib/core/common/OreDictionaryRegistrar.kt

```diff
@@ -61,11 +62,17 @@ object OreDictionaryRegistrar {
     @SubscribeEvent(priority = EventPriority.LOWEST)
     fun init(e: RegistryEvent.Register<Item>) {
         toRegister.forEach { stack, key -> OreDictionary.registerOre(key, stack()) }
-        furnaceRecipeStacks.forEach { stack, out -> FurnaceRecipes.instance()
-                .addSmeltingRecipe(stack(), out.first(), out.second)}
-        furnaceRecipeItems.forEach { item, out -> FurnaceRecipes.instance()
-                .addSmelting(item(), out.first(), out.second)}
-        furnaceRecipeBlocks.forEach { block, out -> FurnaceRecipes.instance()
-                .addSmeltingRecipeForBlock(block(), out.first(), out.second)}
+        furnaceRecipeStacks.forEach { (stack, output, xp) ->
+            FurnaceRecipes.instance()
+                .addSmeltingRecipe(stack(), output(), xp)
+        }
+        furnaceRecipeItems.forEach { (item, output, xp) ->
+            FurnaceRecipes.instance()
+                .addSmelting(item(), output(), xp)
+        }
+        furnaceRecipeBlocks.forEach { (block, output, xp) ->
+            FurnaceRecipes.instance()
+                .addSmeltingRecipeForBlock(block(), output(), xp)
+        }
```

Comment:
We should clear the lists/maps after usage (we'll never need them anymore), or even better set their refs to `null` so they can be fully GC'd.